handlers.go

```diff
@@ -12,13 +12,13 @@ import (
 	"github.com/libp2p/go-libp2p-core/peerstore"
 	pstore "github.com/libp2p/go-libp2p-peerstore"
 
```

Comment:
Import prefixes shouldn't be removed, see [this discussion](https://github.com/ipfs/go-ipfs/issues/5056).

Reply:
With the core refactor, we took the opportunity to bring sense to the package name / import path kerfuffle. As a result, we no longer use aliases when importing from core.

However, there shouldn’t be an import for the old and new packages here, I think (on mobile, so limited to check).

Reply:
@raulk Can you point to that discussion? Thanks

Reply:
Also, this comment is pointed at the ipfs/whyrusleeping imports not the libp2p ones affected by the core refactor.

Reply:
That discussion was not on my radar; I just explained the convention and why here: https://github.com/ipfs/go-ipfs/issues/5056#issuecomment-510843457